runtimes/go-1.19/example/main.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+)
+
+/*
+   'req' variable has:
+       'headers' - object with request headers
+       'payload' - object with request body data
+       'variables' - object with function variables
```

Comment:
let's call this stuff just like how it would be used like:

```suggestion
       'Headers' - map with request headers
       'Payload' - string with request body data
       'Variables' - map with function variables
```

Same with `res`.